sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -2160,7 +2173,8 @@ describe("Atom management - Authentication", function(): void {
       userMetadata: "test metadata",
       status: "ReceiveDisabled",
       subscriptionName: managementSubscription1,
-      topicName: managementTopic1
+      topicName: managementTopic1,
+      availabilityStatus: "Unknow"
```

Comment:
I believe
```suggestion
      availabilityStatus: "Unknown"
```

Reply:
"Unknow (v tr.): to remove or otherwise forget previously gained knowledge or insight"

Reply:
@HarshaNalluru Good catch 👍 . Is there a way to run integration tests on my local without setting up the azure resources? When I run `rushx test` under `service-bus` project, I get 
`Error: Define SERVICEBUS_CONNECTION_STRING in your environment before running integration tests.`

Reply:
Not possible at the moment for the service-bus project. You'd need azure resources to run the live tests for service-bus on the local machine.